src/main/kotlin/uk/gov/dluhc/printapi/database/entity/Certificate.kt

```diff
@@ -76,7 +76,7 @@ class Certificate(
      */
     @field:NotNull
     @Enumerated(EnumType.STRING)
-    var status: Status? = null,
+    var status: PrintRequestStatus.Status? = null,
```

Comment:
Can this be added as an import?

Reply:
I _could_ and I kinda thought about it, but I decided against it because then we are back to referencing `Status` in isolation without the context of what kind of status it is.

eg: it would be this:
```
import uk.gov.dluhc.printapi.database.entity.PrintRequestStatus.Status

    @field:NotNull
    @Enumerated(EnumType.STRING)
    var status: Status? = null,
```
I know it would be there in the import, but it felt a bit lost